src/ShaderGen.Build/ShaderGen.targets

```diff
@@ -23,6 +23,7 @@
       <_SGArgs Condition="'$(ListAllShaderPaths)' == 'true'">$(_SGArgs) --listall</_SGArgs>
       <_SGArgs Condition="'$(ShaderProcessorAssembly)' != ''">$(_SGArgs) --processor "$(ShaderProcessorAssembly)"</_SGArgs>
       <_SGArgs Condition="'$(ShaderProcessorArgs)' != ''">$(_SGArgs) --processorargs "$(ShaderProcessorArgs)"</_SGArgs>
+      <_SGArgs Condition="'$(DebugShader)' == 'true' || '$(Configuration)' == 'Debug'">$(_SGArgs) --debug</_SGArgs>
```

Comment:
Somehow I'd forgotten this, but MSBuild actually uses "or" as the operator here. I guess I should be happy that I haven't written much MSBuild lately 😄 .

`Condition="'$(DebugShader)' == 'true' Or '$(Configuration)' == 'Debug'"`

I'm just going to add on top of your branch before merging.